go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/dmitryikh/leaves
+module github.com/kgwinnup/leaves
```

Comment:
please use the path from this repo.

BTW, is it possible to omit github.. path  in every module?

Reply:
Originally, I was using "replace" with gomod, but until/if this gets merged in the only solution I found was to just rename all the paths so I could continue with my current project. I don't know if you can omit the github part, possibly golang has some magic? Regardless, I will change this import as it definitely doesn't make sense in the PR.